.betterer.results

```diff
@@ -928,6 +928,10 @@ exports[`better eslint`] = {
       [0, 0, 0, "Unexpected any. Specify a different type.", "4"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "5"]
     ],
+    "packages/grafana-ui/src/components/Tabs/Tab.tsx:5381": [
+      [0, 0, 0, "Do not use any type assertions.", "0"],
+      [0, 0, 0, "Do not use any type assertions.", "1"]
+    ],
```

Comment:
We don't plan on "fixing" these type assertions, so use eslint ignore comments to silence the linter instead.